build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'com.surveysampling.scm'
+version = '1.0.0-SNAPSHOT'
+
+description = 'Java program for Raspberry Pi (pi4j) to control a continuous integration traffic light via the GPIO pins.'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+     maven { url "http://ctnexus01.surveysampling.com/content/groups/maven" }
```

Comment:
tl;dr `gradle init --type=pom`  uses the active profiles in `~/.m2/settings.xml` when generating `build.gradle`

I had an active profile for SSI that configured the Nexus repository in `~/.m2/settings.xml`. I was scratching my head after looking at commit: 537e84419999166db86f189649e044f0032110ab; wondering out how the Nexus address got in there.